pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -871,4 +872,63 @@ self: super: {
+  utf8-string = hasNoDataOutput super.utf8-string;
+  utf8-light = hasNoDataOutput super.utf8-light;
+  unix-compat = hasNoDataOutput super.unix-compat;
+  unbounded-delays = hasNoDataOutput super.unbounded-delays;
+  transformers-compat = hasNoDataOutput super.transformers-compat;
+  tagged = hasNoDataOutput super.tagged;
+  th-lift = hasNoDataOutput super.th-lift;
+  disk-free-space = hasNoDataOutput super.disk-free-space;
+  ansi-terminal = hasNoDataOutput super.ansi-terminal;
+  ansi-wl-pprint = hasNoDataOutput super.ansi-wl-pprint;
+  appar = hasNoDataOutput super.appar;
+  magic = hasNoDataOutput super.magic;
+  auto-update = hasNoDataOutput super.auto-update;
+  mountpoints = hasNoDataOutput super.mountpoints;
+  base16-bytestring = hasNoDataOutput super.base16-bytestring;
+  mtl = hasNoDataOutput super.mtl;
+  IfElse = hasNoDataOutput super.IfElse;
+  byteorder = hasNoDataOutput super.byteorder;
+  bytestring-builder = hasNoDataOutput super.bytestring-builder;
+  network-info = hasNoDataOutput super.network-info;
+  cabal-doctest = hasNoDataOutput super.cabal-doctest;
+  network-multicast = hasNoDataOutput super.network-multicast;
+  old-locale = hasNoDataOutput super.old-locale;
+  code-page = hasNoDataOutput super.code-page;
+  hslogger = hasNoDataOutput super.hslogger;
+  cmdargs = hasNoDataOutput super.cmdargs;
+  random = hasNoDataOutput super.random;
+  data-default-class = hasNoDataOutput super.data-default-class;
+  stm = hasNoDataOutput super.stm;
+  stm-chans = hasNoDataOutput super.stm-chans;
+  data-default-instances-containers = hasNoDataOutput super.data-default-instances-containers;
+  text = hasNoDataOutput super.text;
+  data-default-instances-dlist = hasNoDataOutput super.data-default-instances-dlist;
+  data-default-instances-old-locale = hasNoDataOutput super.data-default-instances-old-locale;
+  data-default = hasNoDataOutput super.data-default;
+  StateVar = hasNoDataOutput super.StateVar;
+  digest = hasNoDataOutput super.digest;
+  easy-file = hasNoDataOutput super.easy-file;
+  entropy = hasNoDataOutput super.entropy;
+  extensible-exceptions = hasNoDataOutput super.extensible-exceptions;
+  file-embed = hasNoDataOutput super.file-embed;
+  fail = hasNoDataOutput super.fail;
+  generics-sop = hasNoDataOutput super.generics-sop;
+  mime-types = hasNoDataOutput super.mime-types;
+  ghc-paths = hasNoDataOutput super.ghc-paths;
+  mmorph = hasNoDataOutput super.mmorph;
+  haskell-lexer = hasNoDataOutput super.haskell-lexer;
+  old-time = hasNoDataOutput super.old-time;
+  hex = hasNoDataOutput super.hex;
+  pcre-light = hasNoDataOutput super.pcre-light;
+  hostname = hasNoDataOutput super.hostname;
+  prelude-extras = hasNoDataOutput super.prelude-extras;
+  hxt-charproperties = hasNoDataOutput super.hxt-charproperties;
+  hxt-unicode = hasNoDataOutput super.hxt-unicode;
+  primitive = hasNoDataOutput super.primitive;
+  regex-base = hasNoDataOutput super.regex-base;
+  logict = hasNoDataOutput super.logict;
```

Comment:
you would have to read the contents of the cabal file, and potentially the `Setup.hs` if they have modified it

the only other option (which i have temporarily turned back on to skip ahead), is to just make an empty data output always